test/testsuites/invalidopsrctests.go

```diff
@@ -65,7 +65,7 @@ func testOpSrcWithInvalidEndpoint(t *testing.T) {
 
 	// Check that OperatorSource is in "Downloading" state with appropriate message
```

Comment:
This should say `Check that OperatorSource is in "Configuring" state` instead of `Downloading`